update_prevalence.py

```diff
@@ -1362,6 +1362,11 @@ def parse_romania_prevalence_data(cache: DataCache, data: AllData) -> None:
     except pydantic.error_wrappers.ValidationError as e:
         print_and_log_to_sentry(f"Discarding county-level data from Romania due to error: {e}")
         return
+    latest_date = max([region.Date for region in romania_regions])
+    if effective_date > latest_date:
```

Comment:
Should we give a day or two of leeway here?

Reply:
@beshaya: Unfortunately, it's not as easy as adding a day or two to that `if` statement--the script is built with the assumption that we can always join data against the latest date from JHU.   Later on in processing, we blow up if the effective_date is not found -  https://github.com/microCOVID/microCOVID/blob/main/update_prevalence.py#L419-L421 - which is what was stopping updates.
 
Note that this problem doesn't bother us with api.opencovid.ca, as we pass the "fill=true" flag, causing it to just duplicate the most recent day's actual data as needed up to the current date.  Filling is not a fantastic solution when it's out of our control--the site could go indefinitely stale and we'd have no way to know.  Potentially that approach could be used on our side for both countries, and ideally in a way that preserves the real `updatedAt` date, but I'd prefer to focus on getting the updates running again in this PR.

Reply:
Ok, fair enough. Thanks for thinking this through. I agree that getting updates running again is top priority.